doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -102,6 +102,15 @@ raycluster-sample-2-default-group-vnkkc     2      0      0      4Gi      rayclu
 raycluster-sample-2-head-pwsrm              2      0      0      4Gi      raycluster-sample-2   head     headgroup       3m51s
 ```
 
+You can scale a cluster's worker group like so.
+
+```shell
+$ kubectl ray scale cluster raycluster-sample \
+  --worker-group default-group \
+  --replicas 2
+Scaled worker group default-group in Ray cluster raycluster-sample in namespace default from 1 to 2 replicas
+```
+
```

Comment:
Would you mind adding a command that allows users to verify whether the command works as expected? For example, they could run `kubectl ray get workergroup` to check the default group's REPLICAS.

Reply:
yup, done